cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -1781,6 +1781,7 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
       // Necessary for reading the correct LocalStorage values as we do
       // the initial page load on the beforeEach hook
       cy.reload();
+      cy.wait(500);
```

Comment:
I think this can be avoided by making the assertion below retry correctly. similar to what we do in [richtext.expectValue](https://github.com/contentful/field-editors/blob/9d29a7aab8f073b98f29bde56c41f2615c2be51e/cypress/integration/rich-text/RichTextPage.ts#L104-L106). The difference being we will get a fresh value of `window.actions` every time cypress retry the `.should` command.

You can handle this in the follow up PR

Reply:
Will do